package.json

```diff
@@ -90,5 +95,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
Do we need the sha here?

Reply:
After updating my env if this field is not defined then it will be defined automatically by yarn during install and it appends the sha itself. So because of this I left it as is but I don't think it's strictly necessary is it, I will remove it as it's but present in repos using V4 either.

Reply:
I am not sure about the proper course to chose as we do not use yarn 4 extensively in our repositories. As long as it works and that it is supported I am fine with it, but I think it shall work with only `yarn@1.22.22`